pkg/api/keypair/schema.go

```diff
@@ -21,10 +20,8 @@ const (
 func RegisterSchema(scaled *config.Scaled, server *server.Server, options config.Options) error {
 	server.BaseSchemas.MustImportAndCustomize(harvesterv1.KeyGenInput{}, nil)
 	t := schema.Template{
-		ID: "harvesterhci.io.keypair",
-		Store: store.KeyPairStore{
-			Store: proxy.NewProxyStore(server.ClientFactory, nil, server.AccessSetLookup),
-		},
+		ID:    "harvesterhci.io.keypair",
+		Store: proxy.NewProxyStore(server.ClientFactory, nil, server.AccessSetLookup),
```

Comment:
Is this line removable?

Reply:
Yes, test and the API works without it. ce084d3